_data/tagging-status.yml

```diff
@@ -100,11 +100,12 @@
 
  - name: geometry
    type: pkg
-   status: unknown
+   status: supported
+   comments: tagging of rules (showframe, crop marks) untested
 
  - name: glossaries
    type: pkg
-   status: supported
+   status: unknown
```

Comment:
why unknown?

Reply:
It probably doesn't error, but tagging status is rather unclear. Glossary lists/tables probably more or less inherit from normal lists/tables which can work or not, and the \gls commands itself get at most a link connection but nothing else.

Reply:
ok